src/czbenchmarks/tasks/single_cell/perturbation_expression_prediction.py

```diff
@@ -78,7 +76,6 @@ def __init__(
             random_seed (int): Random seed for reproducibility.
         """
         super().__init__(random_seed=random_seed)
-        self.control_name = control_name
```

Comment:
Please rune the integration test when there are changes to top level class interfaces (not run by default with pytest due to pyproject.toml config). This was easy to fix, but every run taeks a while and that matters as we are so close to release. Thank you.

```
pytest tests/test_integration_end_to_end.py -m integration -k test_end_to_end_perturbation_expression_prediction

============================================================== short test summary info ===============================================================
FAILED tests/test_integration_end_to_end.py::test_end_to_end_perturbation_expression_prediction - TypeError: PerturbationExpressionPredictionTask.__init__() got an unexpected keyword argument 'control_name'
```